src/data/fintype/basic.lean

```diff
@@ -659,14 +659,15 @@ by simp [finset.subset_iff, set.subset_def]
 
 @[simp, mono] theorem to_finset_strict_mono {s t : set α} [fintype s] [fintype t] :
   s.to_finset ⊂ t.to_finset ↔ s ⊂ t :=
-begin
-  rw [←lt_eq_ssubset, ←finset.lt_iff_ssubset, lt_iff_le_and_ne, lt_iff_le_and_ne],
-  simp
-end
+by simp only [finset.ssubset_def, to_finset_mono, ssubset_def]
 
 @[simp] theorem to_finset_disjoint_iff [decidable_eq α] {s t : set α} [fintype s] [fintype t] :
   disjoint s.to_finset t.to_finset ↔ disjoint s t :=
-⟨λ h x hx, h (by simpa using hx), λ h x hx, h (by simpa using hx)⟩
+by simp only [disjoint_iff_disjoint_coe, coe_to_finset]
+
+theorem to_finset_compl [decidable_eq α] [fintype α] (s : set α) [fintype s] [fintype ↥sᶜ] :
+  s.to_finsetᶜ = (sᶜ).to_finset :=
+by { ext a, simp }
```

Comment:
This is stated in the opposite direction to the existing lemma of this name!

Reply:
Fixed.